exo-go/src/template_helpers/index.go

```diff
@@ -32,6 +33,8 @@ dependencies:
 services:
   public:
   private:
+
+templates:
```

Comment:
For `exo add-template`, I think we should still allow the user to update the git URL for existing template/submodule since it has already been implemented and it is quite cumbersome to do that manually. I agree on the `application.yml` part. The other two commands were already implemented the way you described.